plugins/block-extension-tooltip/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@blockly/block-extension-tooltip",
+  "version": "0.0.1",
+  "description": "A Blockly block extension that adds support for custom tooltip rendering.",
+  "scripts": {
+    "audit:fix": "blockly-scripts auditFix",
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "dist": "blockly-scripts build prod",
+    "lint": "blockly-scripts lint",
+    "prepublishOnly": "npm run clean && npm run dist",
+    "start": "blockly-scripts start"
+  },
+  "main": "./dist/index.js",
+  "module": "./src/index.js",
+  "unpkg": "./dist/index.js",
+  "author": "Blockly Team",
+  "keywords": [
+    "blockly",
+    "blockly-plugin",
+    "block-extension-tooltip"
+  ],
+  "homepage": "https://github.com/google/blockly-samples/tree/master/plugins/block-extension-tooltip#readme",
+  "bugs": {
+    "url": "https://github.com/google/blockly-samples/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google/blockly-samples.git",
+    "directory": "plugins/block-extension-tooltip"
+  },
+  "license": "Apache 2.0",
+  "directories": {
+    "dist": "dist",
+    "src": "src"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "devDependencies": {
+    "@blockly/dev-scripts": "^1.2.1",
+    "@blockly/dev-tools": "^2.0.5",
+    "blockly": "^3.20200924.4",
+    "typescript": "^4.0.5"
+  },
+  "peerDependencies": {
+    "blockly": ">=3"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://wombat-dressing-room.appspot.com"
+  },
+  "eslintConfig": {
+    "extends": "@blockly/eslint-config"
+  },
+  "engines": {
+    "node": ">=8.17.0"
+  },
+  "browserslist": [
+    "defaults",
```

Comment:
Please confirm that this works on IE 11, or else change this.

Reply:
I can't test on IE because the playground currently doesn't load on there. Removed IE11 from the list in browserslist and filed https://github.com/google/blockly-samples/issues/500